catalog/app/containers/Bucket/PackageDialog/PackageDialog.js

```diff
@@ -287,35 +380,62 @@ export function MetaInput({ schemaError, input, meta, schema }) {
         </Lab.ToggleButtonGroup>
       </div>
 
-      {value.mode === 'kv' ? (
-        <JsonEditor
-          error={error}
-          disabled={disabled}
-          value={parsedValue}
-          onChange={onJsonEditor}
-          schema={schema}
-        />
-      ) : (
-        <M.TextField
-          variant="outlined"
-          size="small"
-          value={value.text}
-          onChange={handleTextChange}
-          placeholder="Enter JSON metadata if necessary"
-          error={!!error}
-          helperText={
-            !!error &&
-            {
-              jsonObject: 'Metadata must be a valid JSON object',
-            }[error]
-          }
-          fullWidth
-          multiline
-          rowsMax={10}
-          InputProps={{ classes: { input: classes.jsonInput } }}
-          disabled={disabled}
-        />
-      )}
+      <div {...getRootProps({ className: classes.dropzone })} tabIndex={undefined}>
```

Comment:
`tabIndex={undefined}`

Don't understand this

Reply:
by default `getRootProps()` has tabIndex=0 and that makes the dropzone focusable with an ugly blue outline, so i unset it